docs/schema/clientwebpartproperties-complextype-apphostwebfeatures.md

```diff
@@ -18,12 +19,9 @@ ms.assetid: 129a6bac-5a23-ee80-3db8-8e06c8401abe
 
 ## Type information
 
-|   |   |
-|---|---|
-| **Namespace**  | `http://schemas.microsoft.com/sharepoint/` |
-| **Schema file**  | apphostwebfeatures.xsd |
-| **Extension base**  | None |
-
+| Namespace | Schema File | Extension Base |
```

Comment:
Please turn this into a 2 table column with suggested headers of "Name/Value" or "Information/Location" or whatever you feel is most appropriate.